sdk/eventhub/event-hubs/samples/runsamples.ts

```diff
@@ -0,0 +1,28 @@
+import * as dotenv from "dotenv";
```

Comment:
Nit: I think we need the copyright here.

Reply:
Indeed!